src/Microsoft.DotNet.Wpf/src/WpfGfx/core/resources/marshal_generated.cpp

```diff
@@ -3189,7 +3189,7 @@ HRESULT CMilPixelShaderDuce::ProcessUpdate(
     RRETURN(hr);
 }
 
-HRESULT CMilPixelShaderDuce::RegisterNotifiers(CMilSlaveHandleTable *pHandleTable)
+HRESULT CMilPixelShaderDuce::RegisterNotifiers(__in_ecount(1) CMilSlaveHandleTable *pHandleTable)
```

Comment:
I needed to add `__in_ecount(1)` because some definitions of `RegisterNotifiers` had it and others didn't. I decided to add it to all instead of removing it to all. `__in_ecount(1)` is annotations used for build-time validations and shouldn't change behavior at runtime.